### PR TITLE
update docs to match style guide

### PR DESCRIPTION
**Pull request by a-mccarthy**
Signed-off-by: Abigail McCarthy <mabigail@vmware.com>

### CODE REVIEW

File: site/content/docs/main/website-guidelines.md [hunk 1]
```diff
@@ -13,21 +13,21 @@ At the root of the project, run:
...
 ```
 
-This runs all the Ruby dependencies in a container.
+This runs all the Hugo dependencies in a container.
 
 Alternatively, for quickly loading the website, under the `velero/site/` directory run:
 
 ```bash
-bundle exec jekyll serve --livereload --future
+hugo serve
```
Comment (ashish-amarnath) [inline site/content/docs/main/website-guidelines.md:21]: should we change this to `make serve-docs`?

File: site/content/docs/main/website-guidelines.md [hunk 2]
```diff
@@ -13,21 +13,21 @@ At the root of the project, run:
...
 ```
 
-For more information on how to run the website locally, please see our [jekyll documentation](https://github.com/vmware-tanzu/velero/blob/main/site/README-JEKYLL.md).
+For more information on how to run the website locally, please see our [Hugo documentation](https://gohugo.io/getting-started/).
 
 ## Adding a blog post
 
-The `author_name` value must also be included in the tags field so the page that lists the author's posts will work properly (Ex: https://velero.io/tags/carlisia%20campos/). 
+The `author_name` value must also be included in the tags field so the page that lists the author's posts will work properly (Ex: https://velero.io/tags/carlisia%20campos/).
```
Comment (ashish-amarnath) [inline site/content/docs/main/website-guidelines.md:28]: "for example"

File: site/content/docs/main/website-guidelines.md (hunk 1 above)
Comment (a-mccarthy) [reply site/content/docs/main/website-guidelines.md:21]: Do you mean like the example above? I was going for providing thee make command and hugo command, as it was before with the make command and Jekyll build command.

Comment (ashish-amarnath) [reply site/content/docs/main/website-guidelines.md:21]: I didn't see that. Thanks for point that out. You can ignore this comment.

File: site/content/docs/main/website-guidelines.md (hunk 2 above)
Comment (a-mccarthy) [reply site/content/docs/main/website-guidelines.md]: @ashish-amarnath thanks for pointing this out. I actually realized that the instructions were out of date and updated them for the new hugo way of adding a blog

**Pull request merged**